pyproject.toml

```diff
@@ -21,6 +21,6 @@ keywords = [
 [tool.poetry.dependencies]
 # Python 3.7 and later aren't working due to discord.py < 1.0.0 being used
 # See <https://github.com/Rapptz/discord.py/issues/1396>
-python = "^3.5, < 3.7"
-"discord.py" = "^0.16"
+python = "^3.5.3, < 3.8"
```

Comment:
If Python 3.8 can run the bot, feel free to remove the upper version range check :slightly_smiling_face:

Reply:
Comment is removed.
Regarding py 3.8: I will check if it works after I have the bot running again with discord.py 1.2